python/paddle/fluid/tests/unittests/test_imperative.py

```diff
@@ -97,8 +89,9 @@ def test_layer_in_out(self):
     def test_mlp(self):
         np_inp = np.array([[1.0, 2.0], [3.0, 4.0]], dtype=np.float32)
         with fluid.imperative.guard():
+            var_inp = fluid.imperative.base.to_variable(np_inp)
             mlp = MLP()
-            out = mlp(np_inp)
+            out = mlp(var_inp)
```

Comment:
we want to support automatic numpy->var

Reply:
yes, this should be changed when zero copy numpy -> var completed